lib/awspec/helper/finder/autoscaling.rb

```diff
@@ -14,6 +14,14 @@ def find_launch_configuration(id)
                                                                 })
         res.launch_configurations.single_resource(id)
       end
+
+      def select_block_device_mapping(id, device_id)
```

Comment:
@k1LoW Thank you for your some comment!! 🙇 

I renamed method name.
and moved the resource getting process to helper/finder/autoscaling.rb.

Please confirm. 🙏